e2e/execution_test.go

```diff
@@ -1,60 +1,32 @@
 package main
 
 import (
-	"context"
 	"sync"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/bank"
 	"github.com/mesg-foundation/engine/execution"
 	"github.com/mesg-foundation/engine/hash"
-	"github.com/mesg-foundation/engine/protobuf/acknowledgement"
-	pb "github.com/mesg-foundation/engine/protobuf/api"
 	"github.com/mesg-foundation/engine/protobuf/types"
+	executionmodule "github.com/mesg-foundation/engine/x/execution"
 	"github.com/mesg-foundation/engine/x/ownership"
 	"github.com/stretchr/testify/require"
 )
 
 func testExecution(t *testing.T) {
 	var (
-		streamInProgress pb.Execution_StreamClient
-		streamCompleted  pb.Execution_StreamClient
-		err              error
-		executorHash     = testRunnerHash
-		executorAddress  = testRunnerAddress
+		executorHash    = testRunnerHash
+		executorAddress = testRunnerAddress
 	)
 
-	t.Run("create stream nil filter", func(t *testing.T) {
-		_, err := client.ExecutionClient.Stream(context.Background(), &pb.StreamExecutionRequest{})
-		require.NoError(t, err)
-	})
-
-	t.Run("create stream", func(t *testing.T) {
-		streamInProgress, err = client.ExecutionClient.Stream(context.Background(), &pb.StreamExecutionRequest{
-			Filter: &pb.StreamExecutionRequest_Filter{
-				ExecutorHash: executorHash,
-				Statuses:     []execution.Status{execution.Status_InProgress},
-			},
-		})
-		require.NoError(t, err)
-		streamCompleted, err = client.ExecutionClient.Stream(context.Background(), &pb.StreamExecutionRequest{
-			Filter: &pb.StreamExecutionRequest_Filter{
-				ExecutorHash: executorHash,
-				Statuses:     []execution.Status{execution.Status_Completed},
-			},
-		})
-		require.NoError(t, err)
-		acknowledgement.WaitForStreamToBeReady(streamInProgress)
-		acknowledgement.WaitForStreamToBeReady(streamCompleted)
-	})
-
-	t.Run("simple execution", func(t *testing.T) {
```

Comment:
i merge this test and the following one because they were doing the same stuff twice